api/migrations/20160119145322-user.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+var async = require('async');
```

Comment:
Not sure we need this dependency as we are supposed to have Koa. Can't we deal with some `co` calls?